flowmachine/flowmachine/core/server/query_schemas/custom_fields.py

```diff
@@ -29,6 +31,12 @@ def __init__(self, required=False, validate=None, **kwargs):
             **kwargs,
         )
 
+    def _deserialize(self, value, attr, data, **kwargs):
```

Comment:
Can we stick a comment in here to say that this is a temporary workaround until the underlying issue mentioned in #1015 is solved and then we can hopefully remove this again?